src/index.js

```diff
@@ -97,78 +109,95 @@ async function bootstrap(state) {
 
   if (state.seq > 0 && state.bootstrapDone === true) {
     await setSettings(mainIndex);
-    log.info('⛷ Bootstrap: done');
+    log.info('☑️   Bootstrap: done');
```

Comment:
same emoji remark, I use it to search in the logs sometimes

Reply:
fair enough